### ISSUE TITLE
Using the render helper breaks static paths (using path helper)

### ISSUE THREAD

**Issue by ixley** [frctl/fractal#137]
Using the `render` helper to pull in a sub-component breaks paths that use the `path` helper by rendering an incorrect static path. 

The example case is a component with an image path. Pulling in a sub-component using a partial include works fine, but when the component gets rendered, the path does not get updated correctly.

``` hbs
// child-component.hbs
<img src={{ path '/assets/img/image.png'}} />

// parent-component.hbs
{{> @child-component }} {{!-- renders correct path <img src='/assets/img/image.png' /> effectively: `http://localhost:3000/assets/img/image.png` --}}

{{ render '@child-component' }} {{!-- renders broken path <img src='assets/img/image.png' /> effectively: http://localhost:3000/components/preview/assets/img/image.png --}}
```

**Comment by ixley** [comment]
Confirmed that this happens with both Handlebars and Nunjucks, so I assume it has to do with the path helper creating a static path before the render helper pulls in those paths.

**Comment by allmarkedup** [comment]
Finally got to this one... fixed in the Handlebars adapter now (v1.1.2) and just looking into the corresponding fix for Nunjucks.
